.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1726,128 +1731,103 @@ stages:
     - template: steps/install-latest-dotnet-sdk.yml
     - template: steps/restore-working-directory.yml
```

Comment:
Does this step download the windows tracer home? I noticed it got removed below

Reply:
> Does this step download the windows tracer home? I noticed it got removed below

Essentially, yes. It restores the working directory, which is uploaded after the tracer home is built, so it's equivalent.